src/app/space/create/deployments/apps/deployment-status-icon.component.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import { By } from '@angular/platform-browser';
+import {
+  Component,
+  DebugElement,
+  Input
+} from '@angular/core';
+
+import { Observable } from 'rxjs';
+import { DeploymentStatusIconComponent } from './deployment-status-icon.component';
+
+import { CpuStat } from '../models/cpu-stat';
+
+const ICON_OK = 'pficon-ok';
+const ICON_WARN = 'pficon-warning-triangle-o';
+const ICON_ERR = 'pficon-error-circle-o';
+const MSG_OK = 'Everything is ok.';
+
+describe('DeploymentStatusIconComponent', () => {
+
+  let component: DeploymentStatusIconComponent;
+  let fixture: ComponentFixture<DeploymentStatusIconComponent>;
+  let stat: CpuStat;
+
+  let mockCpuData = Observable.of({ used: 1, total: 5 } as CpuStat) ;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [  ],
+      declarations: [
+        DeploymentStatusIconComponent
+      ]
+    });
+
+    fixture = TestBed.createComponent(DeploymentStatusIconComponent);
+    component = fixture.componentInstance;
+    component.cpuDataStream = mockCpuData;
+  });
+
+  it('should set the button\'s initial value to ok', () => {
+    expect(component.icon === 'pficon-ok');
```

Comment:
This goes for all of the `expect`s in this test suite - `expect(val === some)` reads a bit weird and won't provide as descriptive messages if it happens to regress and fail in the future compared to `expect(val).toEqual(some)`. Please change these to use `toEqual`, `toBe`, or whatever other appropriate Matcher functions.